model/post.go

```diff
@@ -268,6 +268,14 @@ type GetPostsOptions struct {
 	Direction                string // Only accepts up|down. Indicates the order in which to send the items.
 }
 
+type PostCountOptions struct {
+	// Only include posts on a specific team. "" for any team.
+	TeamId          string
+	MustHaveFile    bool
+	MustHaveHashtag bool
+	ExcludeDeleted  bool
```

Comment:
I suggest inverting this to "IncludeDeleted", I think the convention in our app and store code is to exclude deleted by default, and have an `IncludeDeleted` option.

Reply:
This func **includes** the deleted posts by default. So, that's why I added the `ExcludeDeleted` option to keep the func default behaviour and to filter out for specific cases only.

Reply:
SGTM